base/Dockerfile

```diff
@@ -31,42 +31,50 @@ RUN python3 -m pip install -v --no-cache-dir -U -r requirements.txt
 
 FROM alpine:3.16.1
 LABEL mainainer='b3vis'
-COPY --chmod=755 entry.sh /entry.sh
-RUN apk upgrade --no-cache \
-    && apk add --no-cache \
+ARG PYTHON_VERSION=3.10
+VOLUME /mnt/source
+VOLUME /mnt/borg-repository
+VOLUME /root/.borgmatic
+VOLUME /etc/borgmatic.d
+VOLUME /root/.config/borg
+VOLUME /root/.ssh
+VOLUME /root/.cache/borg
+HEALTHCHECK --interval=30s --timeout=10s --start-period=20s --retries=3 CMD borgmatic --version || exit 1
+RUN apk add --update --no-cache \
     tzdata \
     sshfs \
     python3 \
     openssl \
+    libxxhash \
+    zstd \
     fuse \
     ca-certificates \
+    logrotate \
     lz4-libs \
     libacl \
     postgresql-client \
     mariadb-client \
     mongodb-tools \
     curl \
     findmnt \
-    logrotate \
     bash \
     bash-completion \
-    && rm -rf /var/cache/apk/*
-VOLUME /mnt/source
-VOLUME /mnt/borg-repository
-VOLUME /etc/borgmatic.d
-VOLUME /root/.borgmatic
-VOLUME /root/.config/borg
-VOLUME /root/.ssh
-VOLUME /root/.cache/borg
-COPY --chmod=755 --from=builder /usr/lib/python3.10/site-packages /usr/lib/python3.10/
-COPY --chmod=755 --from=builder /usr/bin/borg /usr/bin/
-COPY --chmod=755 --from=builder /usr/bin/borgfs /usr/bin/
-COPY --chmod=755 --from=builder /usr/bin/borgmatic /usr/bin/
-COPY --chmod=755 --from=builder /usr/bin/generate-borgmatic-config /usr/bin/
-COPY --chmod=755 --from=builder /usr/bin/upgrade-borgmatic-config /usr/bin/
-COPY --chmod=755 --from=builder /usr/bin/validate-borgmatic-config /usr/bin/
-COPY --chmod=755 --from=builder /usr/bin/markdown_py /usr/bin/
-COPY --chmod=755 --from=builder /usr/bin/normalizer /usr/bin/
-COPY --chmod=755 --from=builder /usr/bin/apprise /usr/bin/
+    bash-doc \
+    && rm -rf \
+    /var/cache/apk/* \
+    /.cache
+COPY --from=builder /usr/lib/python${PYTHON_VERSION}/site-packages /usr/lib/python${PYTHON_VERSION}/
+COPY --chmod=755 --from=builder \ 
+                    /usr/bin/borg \
+                    /usr/bin/borgfs \
+                    /usr/bin/borgmatic \
+                    /usr/bin/generate-borgmatic-config \
+                    /usr/bin/upgrade-borgmatic-config \
+                    /usr/bin/validate-borgmatic-config \
+                    /usr/bin/markdown_py \
+                    /usr/bin/normalizer \
+                    /usr/bin/apprise \
+                    /usr/bin/
```

Comment:
Dodgy indentation here bud, the first / should roughly line up with --chmod

Reply:
Rgr that! Resolved.